tools/linter/adapters/cmake_format_linter.py

```diff
@@ -0,0 +1,155 @@
+"""Adapter for https://github.com/cheshirekow/cmake_format."""
+
+from __future__ import annotations
+
+import argparse
+import concurrent.futures
+import logging
+import os
+import subprocess
+import sys
+
+from lintrunner_adapters import (
```

Comment:
hmm, I see that our other linters don't actually rely on lintrunner_adapters. Guess this probably only worked for me locally because I have it installed and this will require modifications?